### ISSUE TITLE
Order of columns in AQL resultset are in arbitrary order

### ISSUE THREAD

**Issue by nr23730** [ehrbase/ehrbase#37]
When I execute AQL queries the columns of the resultset are in an arbitrary order. I would expect them in the order given in the query, as also shown here: https://specifications.openehr.org/releases/ITS-REST/latest/query.html#query-execute-query-post

Example: 
```
{
  "q": "SELECT tnm/data[at0001]/events[at0002]/data[at0003]/items[at0008]/value/value as T, tnm/data[at0001]/events[at0002]/data[at0003]/items[at0009]/value/value as N, tnm/data[at0001]/events[at0002]/data[at0003]/items[at0010]/value/value as M FROM EHR e[ehr_id/value='4b688173-0388-4cf2-9062-694b86b7fbe3'] CONTAINS COMPOSITION c[openEHR-EHR-COMPOSITION.tnm_stage_clinical.v0] CONTAINS OBSERVATION tnm[openEHR-EHR-OBSERVATION.tnm_stage_clinical.v0]"
}
```

Result: 
```
  <columns>
    <columns>
      <path>/data[at0001]/events[at0002]/data[at0003]/items[at0008]/value/value</path>
      <name>T</name>
    </columns>
    <columns>
      <path>/data[at0001]/events[at0002]/data[at0003]/items[at0010]/value/value</path>
      <name>M</name>
    </columns>
    <columns>
      <path>/data[at0001]/events[at0002]/data[at0003]/items[at0009]/value/value</path>
      <name>N</name>
    </columns>
  </columns>
```

Is this intended or an issue?

Thanks in advance!

**Comment by Geisterfalle** [comment]
Thanks for raising this issue. This should not be intended behavior from my perspective. We will fix this soonly.

**Comment by chevalleyc** [comment]
Resultset in XML is basically non standard and there is no provision to support it; that is only JSON result formatting is currently specified. Further, using JSON, since results are referenced in a hash table, order is not relevant and should be resolved at client level.

**Comment by stefanspiska** [comment]
@chevalleyc I agree that the order should be handelt on client level. Hover it make it nicer when a human is working with the endpoint. Since the performance of HashMap and LinkedHashMap are similar I would like to see https://github.com/ehrbase/ehrbase/pull/69 approved.

**Comment by chevalleyc** [comment]
The changes have been merged with PR #69 . Many thanks!
